tests/language-server-integration-tests/src/test/java/org/ballerinalang/langserver/test/completion/CompilerPluginCompletionTests.java

```diff
@@ -35,7 +35,7 @@ public void compilePlugins() {
     }
 
     @Override
-    @Test(dataProvider = "completion-data-provider")
+    @Test(dataProvider = "completion-data-provider", enabled = false)
```

Comment:
Shall we create an issue for this?

Reply:
Created an issue https://github.com/ballerina-platform/ballerina-lang/issues/42226